### ISSUE TITLE
Add Moves, Ability, Held Item to the selection

### ISSUE THREAD

**Issue by Fonikku** [richi3f/pokemon-team-planner#52]
Everything looks good, but need the above additions to make it perfect!
4 Moves by each Pokemon.
Pokemon's Ability.
Held Item of that Pokemon.
All that in a panel under the "shiny" & "Terastalize" icons. Make them toggle-able.

**Comment by SokroKalimi** [comment]
It isn't that kind of Team Builder. This is more to make outlines for a casual run, the Tera Type and Shinyness don't even save past reloads, I doubt the moveset, ability, or held item would either.

**Comment by Dpmt** [comment]
I think abilities should be added at minimum. The difference between a Pokémon having a ground weakness and one having an immunity with levitate for instance can be substantial if you are trying to balance a team using the analysis function.

**Comment by SokroKalimi** [comment]
That's a good point, but some Pokemon, namely the Weezing and Bronzong lines, do not always have Levitate.

**Comment by Dpmt** [comment]
Agreed, Bronzong also can have Heat Proof to nullify its fire weakness.

Here is a list of moves that change type effectiveness the generation they started changing type effectiveness and the Pokémon that have them. I’m not sure how to represent + or - 25% from things like dry skin’s fire weakness or filter/solid rock/prism armor reducing super effective hits by 25% probably best to ignore reductions but include weaknesses.

Dry Skin (Water Immunity, Fire +25%)
Gen IV
- Paras
- Parasect
- Crogunk
- Toxicroak
Gen V
- Jynx (Hidden Ability)
Gen VI
- Helioptile
- Heliolisk

Earth Eater (Ground Immunity)
Gen IX
- Orthworm

Flash Fire (Fire Immunity)
Gen III
- Vulpix
- Ninetails
- Growlithe
- Arcanine
- Ponyta
- Rapidash
- Flareon
- Houndour
- Houndoom
Gen IV
- Heatran
Gen V
- Cyndaquil (Hidden Ability)
- Quilava (Hidden Ability)
- Typhlosion (Hidden Ability)
- Entei (Hidden Ability, Gen V and VI only)
- Litwick
- Lampent
- Chandelier
- Heatmor
Gen VIII
- Rolycoly (Hidden Ability)
- Carkol (Hidden Ability)
- Coalossal (Hidden Ability)
- Sizzlipede
- Centiskorch
Gen IX
- Charcadet
- Armarouge
- Ceruledge
- Hisuian Growlithe
- Hisuian Arcanine

Fluffy (Fire Weakness)
Gen VI
- Stufful
- Bewear
Gen VIII
- Wooloo
- Dubwool
Gen IX
- Greavard (Hidden Ability)
- Houndstone (Hidden Ability)

Heatproof (Fire Resistance)
Gen IV
- Bronzor
- Bronzong
Gen VIII
- Rolycoly

Levitate (Groun Immunity)
Gen III
- Ghastly
- Haunter
- Gengar (until Gen VII)
- Koffing
- Weezing
- Misdreavus
- Unown
- Vibrava
- Flygon
- Lunatone
- Solrock
- Baltoy
- Claydol
- Duskull
- Chimecho
- Latias
- Latios
Gen IV
- Mismagius
- Chingling
- Bronzor
- Bronzong
- Carnivine
- Rotom (All Forms)
- Uxie
- Mespirit
- Azelf
- Giratina (Origin Forme)
- Cresselia
Gen V
- Tynamo
- Eelektrik
- Eelektross
- Cryogonal
- Hydreigon
Gen VI
- Mega Latias
- Mega Latios
Gen VII
- Vikavolt
Gen VIII
- Galarian Weezing

Lightning Rod (Electric Immunity staring Gen V)
Gen V
- Pikachu (Hidden Ability)
- Raichu (Hidden Ablility)
- Cubone
- Marowak
- Rhyhorn
- Rhydon
- Goldeen (Hidden Ablility)
- Seaking (Hidden Ability)
- Zapdos (Hidden Ability, Gen V only, Always Illegal)
- Pichu (Hidden Ability)
- Electrike
- Manectric
- Rhyperior
- Blitzel
- Zebstrika
Gen VI
- Cosplay Pikachu
- Mega Sceptile
- Plusle (Hidden Ability)
Gen VIII
- Alolan Marowak
- Togedemaru
- Pincurchin

Motor Drive (Electric Immunity)
Gen IV
- Electivire
Gen V
- Blitzel
- Zebstrika
- Emolga (Hidden Ability)

Purifying Salt (Ghost Resistance)
Gen IX
- Nacli
- Naclstack
- Garganacl

Sap Sipper (Grass Immunity)
Gen V
- Marill (Hidden Ability)
- Azumarill (Hidden Ability)
- Girafarig (Hidden Ability)
- Stantler (Hidden Ability)
- Miltank (Hidden Ability)
- Azurill (Hidden Ability)
- Blitzel (Hidden Ability)
- Zebstrika (Hidden Ability)
- Deerling
- Sawsbuck
- Bouffalant
Gen VI
- Skiddo
- Gogoat
- Goomy
- Sliggoo
- Goodra
Gen VII
- Drampa
Gen IX
- Hisuian Sliggoo
- Hisuian Goodra
- Wyrdeer (Hidden Ability)
- Farigiraf (Hidden Ability)

Storm Drain (Water Immunity starting Gen V)
Gen V
- Lileep (Hidden Ability)
- Cradily (Hidden Ability)
- Shellos
- Gastrodon
- Finneon
- Lumineon
- Maractus (Hidden Ability)
Gen IX
- Tatsugiri (All Forms) (Hidden Ability)

Thick Fat (Fire and Ice Resistance)
Gen III
- Seel
- Dewgong
- Snorlax
- Marill
- Azumarill
- Miltank
- Makuhita
- Hariyama
- Azurill
- Spoink
- Grumpig
- Spheal
- Sealeo
- Walrein
Gen IV
- Purugly
- Munchlax
Gen V
- Swinub (Hidden Ability)
- Piloswine (Hidden Ability)
- Mamoswine (Hidden Ability)
- Tepig (Hidden Ability)
- Pignite (Hidden Ability)
Gen VI
- Mega Venusaur
Gen VII
- Alolan Rattata (Hidden Ability)
- Alolan Raticate (Hidden Ability)
Gen VIII
- Appletun (Hidden Ability)
Gen IX
- Lechonk (Hidden Ability)
- Oinkologne (Both Genders) ( Hidden Ability)
- Cetoddle
- Cetitan

Volt Absorb (Electric Immunity)
Gen III
- Jolteon
- Chinchou
- Lanturn
Gen V
- Raikou (Hidden Ability, Gen V and VI only)
- Pachirisu (Hidden Ability)
- Thundurus (Therian Forme)
Gen VI
- Minun  (Hidden Ability)
Gen VII
- Zeraora
Gen VIII
- Dracozolt
- Arctozolt
Gen IX
- Pawmo
- Pawmot
- Wattrel
- Kilowattrel

Water Absorb (Water Immunity)
Gen III
- Poliwag
- Poliwhirl
- Poliwrath
- Lapras
- Vaporeon
- Politoed
- Wooper
- Quagsire
- Mantine
Gen IV
- Mamtyke
Gen V
- Chinchou (Hidden Ability)
- Lanturn (Hidden Ability)
- Suicune (Hidden Ability, Gen V and VI only)
- Cacnea (Hidden Ability)
- Cacturne (Hidden Ability)
- Tympole (Hidden Ability)
- Palpitoad (Hidden Ability)
- Seismitoad (Hidden Ability)
- Maractus
- Frillish
- Jellicent
Gen VI
- Volcanion
Gen VII
- Dewpider (Hidden Ability)
- Araquanid (Hidden Ability)
Gen VIII
- Dracovish
- Arctovish
Gen IX
- Paldian Wooper
- Clodsire

Water Bubble (Fire Resistance)
Gen VIII
- Dewpider
- Araquanid

Well-Baked Body (Fire Immunity)
Gen IX
- Dachsbun

Wonder Guard (Immune to non super effective hits)
Gen III
- Shedinja

**Comment by richi3f** [comment]
@Dpmt thanks for compiling the list! Some ability immunities were included in the earlier versions (e.g. Levitate), but I haven't brought that back. I will think of ways to implement that so abilities are change-able for Pokémon that have abilities that do affect their type matchup.

Also as mentioned by @SokroKalimi, moves and held items are out of the scope of this planner. Unfortunately, I do not have plans to add them in the near future.
